catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/email/EmailNotifier.java

```diff
@@ -167,34 +148,21 @@ private void sendEmailForWorkspace(WorkspaceMetacardImpl workspaceMetacard, Long
 
             mimeMessage.setText(emailBody);
 
-            LOGGER.debug("Attempting to send email to mailHost : {}", mailHost);
+            LOGGER.trace("Attempting to send email");
 
-            Transport.send(mimeMessage);
+            smtpClient.send(mimeMessage);
```

Comment:
Looks like this has been addressed in the smtpClient.  This is pushing the onus of the audit to the implementation of the interface.  This seems reasonable in this case.  @stustison does this address your remaining issues?